lib/connect/index.js

```diff
@@ -1,27 +1,25 @@
-'use strict'
-
+const url = require('url')
```

Comment:
Perhaps we could do this everywhere we use a dependency built into node to make things clear. Not sure if this would affect brower compatibility.

```suggestion
const url = require('node:url')
```

Reply:
I just did a try, seems browserify doesn't support `ndoe:` protocol yet :( we can keep this maybe for the rollup refactor if we ever find a way to make that work

Reply:
Reference: https://github.com/browserify/browserify/issues/2019

Reply:
there's no real benefit to using `node:`; just stick with `url`.